In [22]:
options(scipen=100)

## Correlation Test

In [23]:
- H0: temperature and ice cream sales is not correlated.
- H1: temperature and ice cream sales is correlated.

ERROR: Error in parse(text = x, srcfile = src): <text>:3:0: unexpected end of input
1: - H0: 
2: - H1: 
  ^


## Data 

In [27]:
# Sales Figures
temperature  <- c(32.1, 33.5, 34.1, 34.8, 33.4, 31.0, 29.0)
iceCream     <- c(70  , 75  , 82  , 82  , 78  , 68  , 66  )

In [29]:
summary(temperature)
summary(iceCream)

   Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
  29.00   31.55   33.40   32.56   33.80   34.80 

   Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
  66.00   69.00   75.00   74.43   80.00   82.00 

## Correlation Coefficient 

In [30]:
cor(temperature, iceCream, method = "pearson")

[1] 0.9344274

## Correlation Test

In [31]:
cor.test(temperature, iceCream, method = "pearson")


	Pearson's product-moment correlation

data:  temperature and iceCream
t = 5.8667, df = 5, p-value = 0.002041
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.6120687 0.9904955
sample estimates:
      cor 
0.9344274 


    : p-value of the test is 0.002041, which is less than the significance level alpha = 0.05. Therefore, based on the data we can conclude that pre and post are significantly correlated with a correlation cofficient of 0.9344274 and p-value of 0.002041.